grafana/explore-profiles
# chore: accept pyroscope_git_session cookie name

### ✨ Description

**Related issue(s):** https://github.com/grafana/pyroscope-squad/issues/171

Merge after datasources are able to forward new cookie name:
* https://github.com/grafana/grafana-com/pull/11166

We are planning to rename the `GitSession` cookie to a more convenient `pyroscope_git_session`.

### 📖 Summary of the changes

* Start handling cookies with `pyroscope_git_session`.

### 🧪 How to test?
1. Run this in local.
2. Use a local pyroscope datasource as well, and check that GitHub Integration works as is (with the `GitSession` cookie name). 
3. Change backend to send `pyroscope_git_session` instead, and check that it works as well.
